Description: don't assume that mapfiles are available for every architecture; most likely
 only seen with some zero builds.
Forwarded: http://mail.openjdk.java.net/pipermail/zero-dev/2011-July/000386.html

--- openjdk/jdk/make/common/Program.gmk.orig	2011-07-11 08:51:21.132825771 +0000
+++ openjdk/jdk/make/common/Program.gmk	2011-07-11 12:36:55.602676639 +0000
@@ -56,12 +56,14 @@
 program: $(ACTUAL_PROGRAM)
 
 # Work-around for missing processor specific mapfiles
+ifneq (,$(filter $(ARCH), amd64 i586 sparc sparcv9))
 ifndef CROSS_COMPILE_ARCH
   # reuse the mapfiles in the launcher's directory, the same should
   # be applicable to the tool launchers as well.
   FILES_m = $(BUILDDIR)/java/main/java/mapfile-$(ARCH)
   include $(BUILDDIR)/common/Mapfile-vers.gmk
 endif
+endif
 
 include $(JDK_TOPDIR)/make/common/Rules.gmk
 
